src/main/java/pl/cinema/server/application/ObjectMapperConfig.java

```diff
@@ -0,0 +1,30 @@
+package pl.cinema.server.application;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateTimeDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateTimeSerializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Configuration
+public class ObjectMapperConfig {
+
+    @Bean
+    @Primary
+    ObjectMapper objectMapper() {
+        ObjectMapper mapper = new ObjectMapper();
+        JavaTimeModule timeModule = new JavaTimeModule();
+        timeModule.addDeserializer(LocalDateTime.class, new LocalDateTimeDeserializer(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+        timeModule.addSerializer(LocalDateTime.class, new LocalDateTimeSerializer(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+
+        mapper.configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false);
+        mapper.registerModule(timeModule);
+        return mapper;
```

Comment:
tutaj nie jestem pewny czy dobrze to działa, niby wchodzi debugger do beana, ale później w kontrolerze trzeba było też użyć DateFormatera, do weryfikacji

Reply:
w sensie chodzi Ci o application.yml czy coś innego? ogólnie to jest raczej dobry sposób ustawienia object mappera wg. mnie tylko nie wiem czy czegośnie pominąłem lub nie trzeba czegoś jeszcze dodać...

Reply:
tak, chyba w propertiesach coś się da pogmerać